test_main.py

```
@@ -89,3 +158,70 @@
+        cache.set(url, content)
+
+        # Cache should now contain content
+        assert cache.get(url) == content
+
+        # Check that file was created
+        key = cache._get_cache_key(url)
+        assert (Path(temp_dir) / key).exists()
+
+
+@patch("main.requests.Session.get")
+def test_scrape_website_with_cache(mock_get):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_scrape_website_with_cache`

Add return type annotation: `None`

(ANN201)